src/common/hooks/useDocumentVerifier/index.tsx

```diff
@@ -39,14 +40,17 @@ export const handleVerificationFragment = (
 
 export const useDocumentVerifier = (): DocumentVerifier => {
   const {
-    config: { network }
+    config: { network, verifier }
   } = useConfigContext();
+
   const cancelled = useRef(false);
   const [tamperedCheck, setTamperedCheck] = useState(CheckStatus.CHECKING);
   const [issuedCheck, setIssuedCheck] = useState(CheckStatus.CHECKING);
   const [revokedCheck, setRevokedCheck] = useState(CheckStatus.CHECKING);
   const [issuerCheck, setIssuerCheck] = useState(CheckStatus.CHECKING);
   const [overallValidity, setOverallValidity] = useState(CheckStatus.CHECKING);
+  const [issuerName, setIssuerName] = useState(CheckStatus.CHECKING);
```

Comment:
the initial value is weird ... make it looks like it will be an enum based value but I dont think it's the case